Gemfile.lock

```diff
@@ -181,7 +181,7 @@ GEM
       railties (>= 4.1.0)
       responders
       warden (~> 1.2.3)
-    devise_saml_authenticatable (1.5.0)
+    devise_saml_authenticatable (1.6.1)
```

Comment:
Actually, it might be worth bumping this to 1.6.2 - it's a reversion for a bug introduced in 1.6.1